osmaxx/excerptexport/management/commands/purge_old_result_files.py

```diff
@@ -54,10 +54,14 @@ def _remove_old_files(self):
                 export__output_file__file__isnull=False,
             )
             for old_file in old_files:
-                path = old_file.file.path
                 pk = old_file.id
+                success_message = "Output File #{} removed".format(pk)  # default
+                # if a file has never been attached, but still is stale, remove the object.
```

Comment:
This comment seems misplaced. If you want, replace it with a comment directly above `old_file.delete()` saying something like
```python
# Remove object, no matter whether a on-disk file has been attached or not.
```

(Though, the comment could just as well be removed without replacement, I think.)